src/lib/components/app.svelte

```diff
@@ -3,25 +3,58 @@
     createSerialReadStore,
     type SerialStore,
   } from "$lib/stores/serialRead";
+  import { createBluetoothReadStore } from "$lib/stores/bluetoothRead";
   import Configurator from "./configurator.svelte";
   import ConnectionScreen from "./connectionScreen.svelte";
 
   let serialReadStore: SerialStore | null = null;
+  let bluetoothReadStore: SerialStore | null = null;
 
   const openPort = async (baudRate: number) => {
     const port = await navigator.serial.requestPort();
     serialReadStore = createSerialReadStore(port, baudRate);
     await serialReadStore.connect();
-    serialReadStore.subscribe((event) => {
-      if (event.eventType === "DISCONNECT") {
-        serialReadStore = null;
-      }
-    });
+    serialReadStore.subscribe(handleNotification);
+  };
+
+  const openBluetooth = async () => {
+    console.log("openBluetooth");
+
+    try {
+      const device = await (
+        navigator as Navigator & { bluetooth: any }
+      ).bluetooth.requestDevice({
+        filters: [
+          {
+            name: "HLK-LD2410_6F1F", // TODO: make this configurable
```

Comment:
`namePrefix` should work here. The following works for me on my LD2410C. Would be nice to check it works for a LD2410B but I haven't got mine wired up yet.

```suggestion
            namePrefix: "HLK-LD2410",
```